cmd/e2d/app/run.go

```diff
@@ -52,6 +52,12 @@ type runOptions struct {
 	AWSSecretKey       string `env:"E2D_AWS_SECRET_KEY"`
 	AWSRoleSessionName string `env:"E2D_AWS_ROLE_SESSION_NAME"`
 
+	AzureAccountName    string        `env:"E2D_AZURE_ACCOUNT_NAME"`
+	AzureAccountKey     string        `env:"E2D_AZURE_ACCOUNT_KEY"`
+	AzureStorageAccount string        `env:"E2D_AZURE_STORAGE_ACCOUNT"`
```

Comment:
What's the difference between AccountName and StorageAccount? It looks like you're reusing the S3 Bucket name for the ContainerName.

My preference would be to allow using URLs like `https://<accountname>.blob.core.windows.net/<containername>` and have a regex extract the appropriate parts.

Reply:
Rather than specifying a URL like that I reused the SnapshotBackupURL which looks similar except it starts with azure://